website/docs/md_components/progress_components/usage_markdown.md

```diff
@@ -1,9 +1,9 @@
 Progress indicators tell users that more time is needed to process the last user action (for instance, to load a file, or to install a new update), and how much time remains.
 
 ### Do’s
--	Use a progress indicator for any action that takes longer than about 1.0 second
--	Use Looped animation only for fast actions. The animated-spinner icon is an example of this common progress indicator. This indicator should be used for actions that take between 2-10 seconds
--	Percent-done animation: Use for actions that take 10 seconds or more.
+-	use a progress indicator for any action that takes longer than about 1.0 second
+-	use Looped animation only for fast actions. The animated-spinner icon is an example of this common progress indicator. This indicator should be used for actions that take between 2-10 seconds
```

Comment:
if Looped is an identifier add backtics, I don't see why it should be capitalized